src/endpoint/s3/ops/s3_post_object_uploadId.js

```diff
@@ -24,27 +25,53 @@ async function post_object_uploadId(req, res) {
         throw new S3Error(S3Error.MalformedXML);
     }
 
-    const reply = await req.object_sdk.complete_object_upload({
-        obj_id: req.query.uploadId,
-        bucket: req.params.bucket,
-        key: req.params.key,
-        md_conditions: http_utils.get_md_conditions(req),
-        multiparts
-    });
+    // Write periodically to keep the connection alive
+    // TODO: Every complete above the S3_KEEP_ALIVE_WHITESPACE_INTERVAL
+    // will be responded in BODY, we cannot rollback from that for pre HEADER failures
+    // instread of having a fine tuned way start of sending body to make sure that we won't respond by BODY on HEADER failures
+    const interval_body_update = setInterval(() => {
```

Comment:
I would hang this interval on the response object itself, and refactor this function to http_utils, like this:
```js
http_utils.set_keep_alive_whitespace_interval(res);
```

And would implement it like this:
```js
function set_keep_alive_whitespace_interval(res) {
    res.keep_alive_whitespace_interval = setInterval(() => {
        ...
    }, config.S3_KEEP_ALIVE_WHITESPACE_INTERVAL);
    res.keep_alive_whitespace_interval.unref();
    const clear = () => clearInterval(res.keep_alive_whitespace_interval);
    res.on('close', clear);
    res.on('finish', clear);
}
```

and then you don't need to explicitly handle removing the interval in case send_reply is called.